contracts/Vault/VAIVault.sol

```diff
@@ -80,7 +80,7 @@ contract VAIVault is VAIVaultStorage {
      * @notice Claim XVS from VAIVault
      * @param account The account for which to claim XVS
      */
-    function claim(address account) external nonReentrant {
```

Comment:
`external` seems more appropriate because it's not invoked internally. Apart from it, I would keep this change out of this PR if it's not mandatory. We can address these changes in the vaults in the PR https://github.com/VenusProtocol/venus-protocol/pull/236

Reply:
Fixed, [here](https://github.com/VenusProtocol/venus-protocol/pull/224/commits/7864c014204f3060d743e8140b0c943b1c6a0608).